code/go/0chain.net/smartcontract/storagesc/allocation.go

```diff
@@ -171,37 +171,52 @@ func (sc *StorageSmartContract) newAllocationRequestInternal(
 	timings map[string]time.Duration,
 ) (resp string, err error) {
 	m := Timings{timings: timings, start: time.Now()}
-	if err != nil {
-		return "", common.NewErrorf("allocation_creation_failed",
-			"getting blobber list: %v", err)
-	}
 
 	if txn.ClientID == "" {
```

Comment:
when is it possible that transaction does not have clientID? I think we can delete this check